helmfile/charts/networkpolicy-service/values.yaml

```
@@ -0,0 +1,52 @@
+additional: {}
+# |-
+#   kind: NetworkPolicy
+#   apiVersion: networking.k8s.io/v1
+#   metadata:
+#     name: example-np
+#     namespace: default
+#   spec:
+#     policyTypes:
+#       - Ingress
+#       - Egress
+#     podSelector:
+#       matchLabels:
+#         foo: bar
+#     ingress: {}
+#     egress: {}
+harbor:
+  enable: true
+  chartmuseum:
+    httpsIPs:
+      - "0.0.0.0/0" # S3
```

Would it maybe be more clear to rename it to `egressIPs` or something similar? To make it clear that it belongs to egress and not ingress.
```suggestion
  chartmuseum:
    egressIPs:
      - "0.0.0.0/0" # S3
```